src/main/java/com/alkemy/ong/data/gateway/DefaultCategoryGateway.java

```diff
@@ -0,0 +1,59 @@
+package com.alkemy.ong.data.gateway;
+
+import com.alkemy.ong.data.entity.CategoryEntity;
+import com.alkemy.ong.data.repository.CategoryRepository;
+import com.alkemy.ong.domain.model.Category;
+import com.alkemy.ong.domain.gateway.CategoryGateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class DefaultCategoryGateway implements CategoryGateway {
+
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+    @Override
+    public List<Category> findAll() {
+        List<CategoryEntity> entities = categoryRepository.findAll();
+        List<Category> categories = entities.stream().map(this::toModel).collect(Collectors.toList());
+        return categories;
+    }
+
+    @Override
+    public Category findById(Long id) {
+        CategoryEntity categoryEntity = categoryRepository.getById(id);
```

Comment:
q pasa si no existe? Tendría q llegarle al cliente un 404 Resource Not Found. 
laboraste con GlobalExceptionHandler?